FastSimulation/Tracking/python/DetachedTripletStep_cff.py

```diff
@@ -0,0 +1,55 @@
+import FWCore.ParameterSet.Config as cms
+# import the full tracking equivalent of this file
+import RecoTracker.IterativeTracking.DetachedTripletStep_cff
+
+# trajectory seeds
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+detachedTripletStepSeeds = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    simTrackSelection = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.simTrackSelection.clone(
+        skipSimTrackIds = [cms.InputTag("initialStepSimTrackIds")],
+        pTMin = 0.02,
+        maxD0 = 30.0,
+        maxZ0 = 50
+        ),
+    minLayersCrossed = 3,
+    originpTMin = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepSeeds.RegionFactoryPSet.RegionPSet.ptMin,
+    originHalfLength = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepSeeds.RegionFactoryPSet.RegionPSet.originHalfLength,
+    originRadius = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepSeeds.RegionFactoryPSet.RegionPSet.originRadius,
+    layerList = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepSeedLayers.layerList.value()
+    )
+
```

Comment:
after the first iteration of the review,
I'll see whether I can think of a python function to do this import and the edits